.depcheckrc

```diff
@@ -8,5 +8,7 @@ ignores: [
     "@babel/*",
     "@storybook/*",
     "size-limit",
-    "@size-limit/*"
+    "@size-limit/*",
+    "plotly.js",
```

Comment:
Question; why did you need to add plotly to this list? Just wondering...

Reply:
depcheck raises the flag for plotly with an unknown reason though I didn't see any issues. I am assuming it is due to its peer dependency which is react-plotly.